attesters/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+add_subdirectory(nullattester)
```

Comment:
CSV attester and verifier should also be provided.

Reply:
ok, I will add CSV. could you please help compile it under CSV env?